temporalcli/commandsmd/commands.md

```diff
@@ -57,6 +57,43 @@ This document has a specific structure used by a parser. Here are the rules:
 * `--color` (string-enum) - Set coloring. Options: always, never, auto. Default: auto.
 * `--no-json-shorthand-payloads` (bool) - Always all payloads as raw payloads even if they are JSON.
 
+### temporal activity: Complete or fail an activity.
+
```

Comment:
```suggestion

Activity commands operate on [Activity Executions](/concepts/what-is-an-activity-execution).

Activity commands follow this syntax:
`temporal activity [command] [command options]`
```

Just stole it from old CLI, but I would say it's a bit trivial (we hope to come back and clean these)

Reply:
Yes, I'm not in love with that text from the old CLI. I think the help output is pretty clear as-is:

```
$ temporal activity -h
Complete or fail an activity.

Usage:
  temporal activity [command]

Available Commands:
  complete    Complete an activity.
  fail        Fail an activity.
```

and these Markdown links `[Activity Executions](/concepts/what-is-an-activity-execution)` don't link to a valid target (unless I'm missing something?)

Reply:
Not missing anything, I too am not a fan, but I was just leaving many docs things as is to come back and be cleaned up later. Can just leave off too.